tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -51,6 +55,11 @@ def load_model(self, model_name, model_link: str):
             if model_file_name.endswith('.pb'):
                 graph = load_graph(model_file_name)
                 return graph
+        elif is_tf_keras_application(model_link):
+            if model_name == 'ResNet50':
+                return tf.keras.applications.resnet50.ResNet50(weights="imagenet")
```

Comment:
Why do we need to treat ResNet50 differently than other models? Will we need to list other model names similar way here?

Reply:
Agreed to optimize retrieving model by string name in separate PR.